### PR TITLE
[#5] Header and footer

### PR DESCRIPTION
**Pull request by mjordonez2001**
### Issues
- #5 

### What?
- Configures Bootstrap
- Configures React Icons
- Makes skeleton header and footer
- Adds React icons for cart and account page

### Screenshots
Screen Shot 2022-11-26 at 5 26 29 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hearty Capsules</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
   </head>
   <body>
     <div id="root"></div>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
```
Comment (cwmanning) [inline index.html:11]: any idea if you can safely drop this script? I'm not sure what the current state of Bootstrap is, but my hunch is that you may only need the CSS dependency above

File: src/marginals/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function Footer() {
+  return (
+    <nav className="navbar navbar-expand-md bg-light fixed-bottom">
```
Comment (cwmanning) [inline src/marginals/Footer.tsx:3]: `<footer>` here?

**Review by cwmanning — APPROVED** [review]

File: src/pages/Pages.tsx [hunk 3]
```diff
@@ -13,19 +15,25 @@ const Supplement = lazy(async () => await import("./supplement/supplement"));
...
-        <Route path="/login" element={<Login />} />
-        <Route path="/quiz" element={<Quiz />} />
-        <Route path="/shop" element={<Shop />} />
-        <Route path="/supplement" element={<Supplement />} />
-      </Routes>
-    </Suspense>
+    <>
+      <Header />
+      <div className="container">
+        <Suspense fallback={null}>
```
Comment (nick-telsan) [inline src/pages/Pages.tsx:21]: What does this fallback do if it is `null`? Just not render?

**Review by nick-telsan — APPROVED** [review]
👍

File: src/marginals/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom";
+import { SlBag, SlUser } from "react-icons/sl";
+
+function Header() {
+  return (
+    <nav className="navbar navbar-expand-md bg-light sticky-top px-5">
+      <div className="container-fluid">
+        <div className="d-flex col-4">
+          <Link className="nav-link mx-3" to="/shop">
```
Comment (solomonhawk) [inline src/marginals/Header.tsx:9]: React router should have a `NavLink` you can use to optionally style these links with an active state when you're on the corresponding page.

**Review by solomonhawk — APPROVED** [review]

File: src/pages/Pages.tsx (hunk 3 above)
Comment (mjordonez2001) [reply src/pages/Pages.tsx:21]: It doesn't render anything

**Pull request merged**